### DISCUSSION TITLE
MQTT Temperature values

### DISCUSSION THREAD

**Discussion by WirthmU** [aaronwmorris/indi-allsky#1232] [General]
What exactly are the two temperature values that appear in the MQTT export?

**Comment by aaronwmorris** [comment]
What hardware are you using?

**Comment by WirthmU** [comment]
Raspberry Pi 4 Model B Rev 1.2 / 4 GB and Raspberry Pi HQ Kamera.

**Reply by aaronwmorris** [reply]
The first temperature is probably the PI HQ Camera sensor.  The second is the PI4 internal CPU temperature sensor.
